index.html

```diff
@@ -0,0 +1,141 @@
+        <p class="small">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae.</p><br>
+        <a href="#" class="adopt-button">Adopt</a>
+      </article>
+    </div>
+
+    <div class="dog-box">
+      <img class="dog-img" src="images/lucas-card.jpg" alt="A photo of Lucas, an energetic black dog, Shepherd mix.">
+      <article class="dog-profile">
+        <h3>Lucas</h3>
+        <p class="small"><strong>Cost to Adopt:</strong>&nbsp;$123.45</p>
+        <p class="small">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae.</p><br>
+        <a href="#" class="adopt-button">Adopt</a>
+      </article>
+    </div>
+
+    <div class="dog-box">
+      <img class="dog-img" src="images/jake-card.jpg" alt="A photo of Jake on the beach, a happy yellow dog, Lab mix.">
+      <article class="dog-profile">
+      <h3>Jake</h3>
+      <p class="small"><strong>Cost to Adopt:</strong>&nbsp;$123.45</p>
+      <p class="small">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae.</p><br>
+      <a href="#" class="adopt-button">Adopt</a>
+      </article>
+    </div>
+
+    <div class="dog-box">
+      <img class="dog-img" src="images/angus-card.jpg" alt="A photo of Angus in the grass, a happy brown dog, Boxer mix.">
+      <article class="dog-profile">
+        <h3>Angus</h3>
+        <p class="small"><strong>Cost to Adopt:</strong>&nbsp;$123.45</p>
+        <p class="small">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae.</p><br>
+        <a href="#" class="adopt-button">Adopt</a>
+      </article>
+    </div>
+
+    <div class="dog-box">
+      <img class="dog-img" src="images/violet-card.jpg" alt="A photo of Violet, an adorable brown puppy, Lab mix.">
+      <article class="dog-profile">
+        <h3>Violet</h3>
+        <p class="small"><strong>Cost to Adopt:</strong>&nbsp;$123.45</p>
+        <p class="small">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae.</p><br>
+        <a href="#" class="adopt-button">Adopt</a>
+      </article>
+
+    </div>
+  </div>
+
+    <a href="Dogs.html" class="light-button">See More Dogs</a>
+
+  </section>
+
+
+  <footer>
+    <section class="footer-section">
+    <img class="footer-img" src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+    <h3 class="footer-h3">Aborpos nisto inci tet aut que volupti beruptiberro to</h3>
+    <h3 class="footer-h3">ex esseque invenienihil imaximu scideru ptistiusam consequunt</h3>
+    <nav class="footer-nav">
+      <h3 class="footer-h3"><a class="nav" href="index.html">Home</a> &nbsp;/&nbsp; <a class="nav" href="dogs.html">Dogs</a> &nbsp;/&nbsp; <a class="nav" href="blog.html">Blog</a></h3>
```

Comment:
It's nice to have these stacked vertically in the code. Maybe instead of wrapping all of them in an H3, just style the links themselves?